meson_options.txt

```diff
@@ -0,0 +1,5 @@
+option('driverdir', type : 'string', description : 'drivers path')
+option('disable_x11', type : 'boolean', value : false)
+option('with_wayland', type : 'combo', choices : ['yes', 'no', 'auto'], value : 'auto')
```

Comment:
Can you make x11 and wayland options consistent.  For example, `with_x11` and `with_wayland` and have same combo options.